docs/source/conventions.md

```diff
@@ -73,6 +73,41 @@ Mixing singular and plural is confusing.
 For example, using `/users` for retrieving users, and `/user/21` to retrieve a single user.
 
 
+### URL Parameters (singular vs plural)
+
+**Rule:** Use plural for URL parameters that can contain one or multiple items. Use singular for parameters that can contain only one single item.
+
+**Do:**
+
+If the parameter allows passing multiple values:
+
+```text
+/tokens
+```
+
+If the parameter allows passing a single value only:
+
+```text
+/token
+```
+
+**Don't:**
+
+If the parameter allows passing multiple values:
+
+```text
+/token
+/token_list
+/token_array
+/token_set
+```
+
+**Reason:**
+
+The naming should clearly indicate if an attribute expects a single (singular) item or multiple items (plural). We decided to use plural instead of appending "_list" or "_array" or "_set".
```

Comment:
Uhm, please check:
- https://docs.quantifiedcode.com/python-anti-patterns/readability/putting_type_information_in_a_variable_name.html
- https://stackoverflow.com/questions/111933/why-shouldnt-i-use-hungarian-notation

Reply:
@ale-rt thanks for the pointers. Though, we do NOT recommend to use "_array", "_set" and stuff. Maybe we have to work on the wording if that paragraph is misleading. :)

Feel free to improve the wording or tell me if you object to the decision to go with plural for parameters that accept multiple parameters.

Reply:
I am not objecting! 
And I do not like to waste our time on philosophical digression on how variables should be name.
I am just pointing out that other people did that and concluded that `token_list` was a bad variable name.

With that said, you can use it anyway, nobody will sue you!

Sorry for the noise and have a great sprint!

Reply:
I had to read that paragraph twice myself. @ale-rt I think we both got confused. The intent is "Do not append the type to the name of the variable".

Let me take a crack at rewording this so that it does not trip up other people, too.

Reply:
Oh, yeah... sorry

Reply:
No need to apologize. It threw me for a loop, too. 🤪

Reply:
@ale-rt @stevepiercy thank you both for your feedback! The fact that you misunderstood the paragraph shows that we need to improve it. I am open to suggestions here. :)

Reply:
Just scrolled down. :) Thanks @stevepiercy!